docs/pywbemcligeneraloptions.rst

```diff
@@ -61,166 +70,470 @@ command, general the options redefinition's are not retained between command
+      $ eval $(pywbemcli -s http://localhost -u username -d root/cimv2)
+
+      $ env |grep PYWBEMCLI
+      export PYWBEMCLI_SERVER=http://localhost
+      export PYWBEMCLI_NAMESPACE=root/cimv2
+
+      $ pywbemcli server namespaces
+      . . . <list of namespaces for the defined server>
+
+.. _`--timeout general option`:
+
+--timeout general option
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The value of ``--timeout\-t`` general option is an integer that defines the
+client side timeout in seconds. The pywbem client includes a timeout mechansim
+that closes a WBEM connection if there was no activity for time period defined
+by this value. Setting the value to zero. Pywbemcli defaults to a predefined
+timeout (normally 30 seconds) if this option is not defined.
+
+
+.. _`--no-verify general option`:
+
+--no-verify general option
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If the ``--no-verify/-n`` boolean general options is set, theclient does not
+verify any certificates received from the WBEM server. Any certificate returned
+by the WBEM server is accepted.
+
+
+.. _`--certfile general option`:
+
+--certfile general option
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The value of the ``--certfile`` is a file path for a X.509 client certificate
+to be presented to the WBEM server along with the ``--keyfile`` option during
+the TLS/SSL handshake. This parameter used only with HTTPS.  If None, no client
+certificate is presented to the server, enabling 1-way authentication.
+Otherwise, the client certificate is presented to the server, enabling 2-way
+(mutual) authentication. This and the ``--keyfile`` are presented to the pywbem
+:class:`pywbem.WBEMConnection` as a single parameter ``X-509``:
+
+* ``--cert_file`` - The file path of a file containing an X.509 client
+  certificate.
+* ``key_file`` - The file path of a file containing the private key belonging
+  to the public key that is part of the X.509 certificate file.
+
+For more information on authentication types, see:
+:https://pywbem.readthedocs.io/en/stable/client/security.html#authentication-types
+
+.. _`--keyfile general option`:
+
+----keyfile general option
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The value of the ``--keyfile`` general option is a file path of the
+the client private key file containing the private key belonging to the public
+key that is part of the X.509 certificate. See ``--certfile`` for more
+information. Not required if the private key is part of the file defined in
+``--certfile``. Not allowed if ``--certfile`` option not provided. Default: No
+client key file. Client private key should then be part of the file defined by
+``--certfile``.
+
+The pywbemcli command line options are as follows (The following complements
+the exact help text output shown in section :ref:`pywbemcli Help Command
+Details`)
+
+
+.. _`--ca-certs general option`:
+
+----ca-certs general option
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The value of the ``--ca-certs`` a file or directory containing certificates
```

Comment:
it is the path name of a file or directory.

Reply:
DONE